### ISSUE TITLE
The extension prevents the C# dev kit extension to work

### ISSUE THREAD

**Issue by aldelaro5** [AvaloniaUI/Avalonia-VSCode-Extension#2]
Type: <b>Bug</b>

Have the C#, C# dev kit and C# intelicode extensions installed alongside the avalonia one. From there, open a project with avalonia (this doesn't affect projects without avalonia).

From there, observe the solution view not appearing.

Extension version: 0.0.5
VS Code version: Code 1.80.2 (2ccd690cbff1569e4a83d7c43d45101f817401dc, 2023-07-27T20:41:52.615Z)
OS version: Linux x64 6.4.7-arch1-1
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-9700K CPU @ 3.60GHz (8 x 4600)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|2, 3, 3|
|Memory (System)|31.21GB (8.97GB free)|
|Process Argv|--unity-launch /home/aldelaro5/vm-shared/Ikemen_GO-v0.99.0-rc.1-linux/save/config.json --crash-reporter-id 11cb79d6-68a2-4e25-a18f-a22d4541ecf4|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|cinnamon|
|XDG_CURRENT_DESKTOP|X-Cinnamon|
|XDG_SESSION_DESKTOP|cinnamon|
|XDG_SESSION_TYPE|x11|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30410667
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyone:30548225
vscccc:30803845
2e4cg342:30602488
f6dab269:30613381
showlangstatbar:30737416
vsctsb:30748421
03d35959:30757346
pythonfmttext:30731395
pythoncmv:30756943
fixshowwlkth:30771522
showindicator:30805244
pythongtdpath:30769146
i26e3531:30792625
gsofa:30804715
pythonnosmt12:30797651
pythonidxpt:30805730
pythonnoceb:30805159
e537b577:30795824
dsvsc013:30795093
dsvsc014:30804076

```

</details>

**Comment by prashantvc** [comment]
Could you be more elaborate about what does not work? Does solution view disappear?

I am not able to reproduce it, the C# DevKit continue to work as designed with Avalonia extension.
<img width="1488" alt="image" src="https://github.com/AvaloniaUI/Avalonia-VSCode-Extension/assets/328076/80b3382c-2691-493e-93ba-21857396638d">

**Comment by aldelaro5** [comment]
On my end at least, what happens is I open any avalonia contained project and I don't really get any meaningful feedback the the C# dev kit is working in general. Normally, when I open a folder, after a few seconds, I see the solution view appear on the left as a different view than the files view alongside other things like the play button on the top right. This works on my end for every projects without the avalonia extension installed.

However, as soon as I install the extension, it changes the behavior with projects containing avalonia. On those projects, C# dev kit doesn't seem to work at all. I check the different output consoles, but nothing meaningful, it just seems to not load in general and I don't get any feedbacks from either the C# dev kit or even the avalonia extension for that matter (I worked on a previously built project, but still can't see a previewer). It gives me the impression it silently crashed and took down C# dev kit with it.

Also indicative of this is when I open a project NOT containing avalonia where it works regardless if I have the extension or not. To prove this was the extension's fault, as soon as I uninstalled it, C# dev kit started to work again.
